sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/LinkedEntityMatch.java

```diff
@@ -51,4 +79,22 @@ public double getConfidenceScore() {
     public String getText() {
         return this.text;
     }
+
+    /**
+     * Get the offset of entity text.
```

Comment:
@maririos made this comment on my PR too, but it would be clearer if this was "Get the offset of the linked entity match text"